app/controllers/facility_journals_controller.rb

```diff
@@ -29,6 +29,13 @@ def index
   def new_with_search
     set_default_variables
     @layout = "two_column_head"
+
+    respond_to do |format|
+      format.csv do
+        @order_details = @order_details.expired_account
```

Comment:
I feel like a comment here might be helpful since we are doing something a little funky.